homeassistant/components/airzone/sensor.py

```diff
@@ -84,12 +84,12 @@ def __init__(
         self._attr_unique_id = f"{entry.entry_id}_{system_zone_id}_{description.key}"
         self.entity_description = description
 
+        if description.key == AZD_TEMP:
+            unit = TEMP_UNIT_LIB_TO_HASS.get(self.get_zone_value(AZD_TEMP_UNIT))
+            if unit != TEMP_CELSIUS:
```

Comment:
I don't understand the check.

Reply:
Removed in 2060696e09bbf0f6374559024ccedd70cf500849

Reply:
Removed in 158c9b5c15592ab3b3df33a0eac7b8059179c680